notification/endpoint/endpoint.go

```diff
@@ -20,13 +20,11 @@ var typeToEndpoint = map[string](func() influxdb.NotificationEndpoint){
 	HTTPType:      func() influxdb.NotificationEndpoint { return &HTTP{} },
 }
 
-type rawJSON struct {
-	Type string `json:"type"`
-}
-
 // UnmarshalJSON will convert the bytes to notification endpoint.
 func UnmarshalJSON(b []byte) (influxdb.NotificationEndpoint, error) {
-	var raw rawJSON
+	var raw struct {
+		Type string `json:"type"`
+	}
```

Comment:
This strict can be helpful to a lot types, can u move it back?

Reply:
it could be, but it isn't atm. Feels awkward to me to pollute the global namespace with something that is never used anywhere else 🤷‍♂. If something wants to use it, they are welcome to abstract it then you know. Main reason I axed it.